packages/teams-js/src/public/monetization.ts

```diff
@@ -68,19 +75,29 @@ export namespace monetization {
   ): Promise<void> {
     let callback: ((error: SdkError | null) => void) | undefined;
     let planInfo: PlanInfo | undefined;
+    // eslint-disable-next-line strict-null-checks/all
```

Comment:
@AE-MS Is this the line we talked yesterday that should be removed? (Or shouldn't be existed in TJS codebase?)

Reply:
If you can remove it and nothing breaks, then totally remove it. If I did it right, these errors should be caught by the tsc compiler now and so disabling the eslint check should not have any effect (and thus it could be removed). Although you don't have to do it as part of this PR, I can just go through a make a single pass across the codebase to remove them.